books/english-composition/bake

```diff
@@ -52,7 +52,9 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :english_composition) do |doc|
   book.chapters.each do |chapter|
     chapter.pages.each do |page|
       page.search('div[data-type="media"]').each do |media_container|
-        BakeIframes.v1(outer_element: media_container)
+        if !media_container.parent[:'data-type'] == 'note'
+          BakeIframes.v1(outer_element: media_container)
+        end
```

Comment:
I have added this condition to prevent double iframes baking when they are places in a note and  in div[data-type="media"] only in the content.